src/Quartz/Impl/AdoJobStore/AdoJobStoreUtil.cs

```diff
@@ -28,17 +29,30 @@ namespace Quartz.Impl.AdoJobStore
 	/// <author>Marko Lahma (.NET)</author>
 	public static class AdoJobStoreUtil
 	{
-	    /// <summary>
-		/// Replace the table prefix in a query by replacing any occurrences of
-		/// "{0}" with the table prefix.
-		/// </summary>
-		/// <param name="query">The unsubstituted query</param>
-		/// <param name="tablePrefix">The table prefix</param>
-		/// <param name="schedNameLiteral">the scheduler name</param>
-		/// <returns>The query, with proper table prefix substituted</returns>
-		public static string ReplaceTablePrefix(string query, string tablePrefix, string schedNameLiteral)
-		{
-			return string.Format(CultureInfo.InvariantCulture, query, tablePrefix, schedNameLiteral);
-		}
-	}
+        /// <summary>
+        /// Replace the table prefix in a query by replacing any occurrences of
+        /// "{0}" with the table prefix.
+        /// </summary>
+        /// <param name="query">The unsubstituted query</param>
+        /// <param name="tablePrefix">The table prefix</param>
+        /// <param name="schedNameLiteral">the scheduler name - no longer required, scheduler name is now a sql parameter</param>
+        /// <returns>The query, with proper table prefix substituted</returns>
+        [Obsolete("Use overload with schedNameLiteral, scheduler name is now a sql parameter")]
+        public static string ReplaceTablePrefix(string query, string tablePrefix, string schedNameLiteral)
```

Comment:
I opted to maintain backward compatibility for any public or protected methods - they should all be marked as obsolete.